src/core/title.vala

```diff
@@ -2,4 +2,5 @@
 
 public interface Games.Title : Object {
 	public abstract string get_title () throws Error;
+	public abstract string get_raw_title () throws Error;
```

Comment:
This doesn't look like something we want: when we want the title, we want the title and no more, if you want to extract some other info like the disc number from a filename, then you should have some " it has nothing to do with the title which should be left untouched.